src/components/profile/actions.js

```diff
@@ -0,0 +1,30 @@
+import * as actionTypes from './actionTypes';
+
+export const getUserProfile = () => ({
+	type: actionTypes.GET_USER_PROFILE,
+});
+
+export const getUserProfileSuccess = data => ({
+	type: actionTypes.GET_USER_PROFILE_SUCCESS,
+	payload: data,
+});
+
+export const getUserProfileFail = data => ({
+	type: actionTypes.GET_USER_PROFILE_FAIL,
+	payload: data,
+});
+
+export const updateUserProfile = (data, close) => ({
+	type: actionTypes.UPDATE_USER,
+	payload: { data, close },
+});
+
+export const updateUserProfileSuccess = data => ({
+	type: actionTypes.UPDATE_USER_SUCCESS,
+	payload: data,
+});
+
```

Comment:
try to practice DRY-do not repeat yourself
you can simply destructure `const {UPDATE_USER_SUCCESS, UPDATE_USER, ...} = actionTypes;`